src/components/explorer/IndividualOverview.js

```diff
@@ -49,19 +45,21 @@ IndividualOverview.propTypes = {
 };
 
 function getAge(individual) {
```

Comment:
should this use the time element component instead?

Reply:
Function was removed in favor of TimeElement component